logging/google/cloud/logging/_gax.py

```diff
@@ -211,13 +212,22 @@ def sink_create(self, project, sink_name, filter_, destination):
         :type destination: str
         :param destination: destination URI for the entries exported by
                             the sink.
+
+        :type unique_writer_identity: bool
```

Comment:
Just a general question whenever I see a 'bool' parameter: Is there any scenario you can image where you might want a third choice?  In which case, make this an enum instead of bool?

Reply:
@duggelz In this case, the parameter corresponds to [a protobuf field][1] so the mapping is as expected. (Worth noting that `proto3` has no distinction between "unset" and "zero value" for simple fields like `bool`, `int`, etc.)

[1]: https://github.com/googleapis/googleapis/blob/c44c8d7edcb75dcafdf75d0cdfa7f98747982266/google/logging/v2/logging_config.proto#L258

Reply:
@duggelz Thank you for reviewing my code.  As @dhermes mentioned, I grabbed the type from the protobuf field.